examples/rollbar-spring-webmvc/src/main/java/com/example/springwebmvc/initializer/AppInitializer.java

```diff
@@ -0,0 +1,33 @@
+package com.example.springwebmvc.initializer;
+
+import com.example.springwebmvc.config.RollbarConfig;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRegistration;
+
+import org.springframework.web.WebApplicationInitializer;
+import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
+import org.springframework.web.servlet.DispatcherServlet;
+
+
+
+
+public class AppInitializer implements WebApplicationInitializer {
+
+  @Override
+  public void onStartup(ServletContext servletContext) throws ServletException {
+    AnnotationConfigWebApplicationContext ctx = new AnnotationConfigWebApplicationContext();
+    ctx.register(RollbarConfig.class);
+    ctx.setServletContext(servletContext);
+    ctx.refresh();
```

Comment:
I think this is not needed

Reply:
the .refresh is needed actually so that the RollbarConfig gets registered or else it doesn't get called

Reply:
from the docs:

> Note that AbstractApplicationContext.refresh() must be called in order for the context to fully process the new classes.